src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,49 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = (
+  {
+    post: {
+      title, body, user, comments,
+    },
+  },
+) => (
+  <div className="PostInfo event">
+    <div className="content">
+      <div className="summary">
+        <h3 className="PostInfo__title">
+          {title}
+        </h3>
+        <span>
+          {'posted by: '}
+        </span>
+        {user
+          ? <UserInfo user={user} />
+          : 'anonymous'}
+      </div>
+
+      <div className="extra text">
+        {body}
+      </div>
+
+      <div className="meta">
+        <a className="comment" href="#comment">
+          <i className="comment icon" />
+          {!comments.length ? (
+            <span data-cy="NoCommentsMessage">No comments</span>
+          ) : (
+            `${comments.length} comments`
+          )}
```

Comment:
not the best code formatting, each element of ternary operation should be in the new line
Also, it's better to check with `> 0` for length